_layouts/default.html

```diff
@@ -9,6 +9,7 @@
   <link href="https://fonts.googleapis.com/css2?family=Google+Sans:wght@300;400;500;700" rel="stylesheet">
   <link href="https://fonts.googleapis.com/css2?family=Google+Sans+Text:wght@300;400;500;700" rel="stylesheet">
   <link href="https://fonts.googleapis.com/css2?family=Public+Sans:wght@600&family=Roboto&family=Material+Icons&display=swap" rel="stylesheet">
+  <script src="https://datacommons.org/datacommons.js"></script>
```

Comment:
Would prefer to keep this on the web component pages only

Reply:
Good callout! Added liquid template logic so this only gets added on web component pages.